Content.Server/Medical/CrewMonitoring/CrewMonitoringConsoleComponent.cs

```diff
@@ -16,4 +17,34 @@ public sealed partial class CrewMonitoringConsoleComponent : Component
     /// </summary>
     [DataField("sensorTimeout"), ViewVariables(VVAccess.ReadWrite)]
     public float SensorTimeout = 10f;
+
+    // DeltaV - start of alert system code
+    /// <summary>
+    ///     Should the component beep if someone goes critical or dies
+    /// </summary>
+    [DataField]
+    public bool AlertsEnabled = true;
+
+    /// <summary>
+    ///     Track sensors that have triggered the crew member critical alert.
+    /// </summary>
+    public HashSet<string> AlertedSensors = [];
+
+    /// <summary>
+    ///     Accumulated time for tracking alert cooldown
+    /// </summary>
+    public TimeSpan AccumulatedTime = TimeSpan.Zero;
```

Comment:
still need to change the logic from adding to a `[DataField(customTypeSerializer: typeof(TimeOffsetSerializer))] public TimeSpan NextAlert;` or something, comparing _timing.CurTime to it

along with `[AutoGenerateComponentPause]` and `[AutoPausedField]`